gbpservice/nfp/service_plugins/firewall/nfp_fwaas_plugin.py

```diff
@@ -0,0 +1,22 @@
+from oslo_config import cfg
+
+from gbpservice.nfp.config_orchestrator.agent import topics
+import neutron_fwaas.extensions
```

Comment:
RPM, did you copy this file on the integration setup and tested? If so, we can merge this. Lets discuss.